tests/client/test_users.py

```diff
@@ -93,3 +93,15 @@ def test_user_delete_user(owner: "ServerUser") -> None:
     user.delete()
     with pytest.raises(ValueError, match="doesn't exist in Argilla"):
         user.delete()
+
+
+def test_print_user(owner: "ServerUser"):
+    import argilla as rg
+
+    rg.init(api_key=owner.api_key)
```

Comment:
```suggestion
    ArgillaSingleton.init(api_key=owner.api_key)
```

Reply:
same here

Reply:
In that case, we should add a test to validate that the main Argilla functions are accessible from the main client module. Imagine a new `rg.pentola_me` mapping to an internal `argilla.client.init.Argilla.pentola_me` method. If we use the direct method, we never know if the exposed `rg.pentola_me` function is really working.